Models/userData.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "name": "Jane",
+    "id": 133245,
+    "email": "jane@hotmail.com",
+    "password": "apple329",
+    "resume_URL": "https://www.sometext.com",

```

Comment:
resume_URL is set to be unique, make sure to change the seed file accordingly
e.g. 
https://www.sometext1.com
https://www.sometext2.com 
etc